src/plugins/intel_gna/src/log/dump.cpp

```diff
@@ -440,11 +440,38 @@ inline void DumpCharArray(std::ostream& dumpFile, const char* carray, size_t cou
     dumpFile << "\n";
 }
 
+class Gna2Stats {
```

Comment:
```suggestion
class GnaMemStats {
```

Reply:
I suggest to change the name to avoid confusion with some performance or other statistics available from GNA library and also with this class being related to GNA library headers (which use the Gna2 prefix).